integration/rest/src/main/java/it/gov/pagopa/rtd/transaction_filter/connector/model/FileMetadata.java

```diff
@@ -23,7 +28,13 @@ public class FileMetadata {
 
   @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
   @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+  @JsonSerialize(using = LocalDateTimeSerializer.class)
+  @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
   @NotNull
   private LocalDateTime transmissionDate;
 
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  private AggregatesDataSummary aggregatesDataSummary;
```

Comment:
the POJO defined in this way enforce a strict parse. Any future changes on the backend model requires to change this POJO and release a new version of batch service. 
Is there any alternative to avoid this situation?